Framework/API/src/MatrixWorkspace.cpp

```diff
@@ -83,6 +84,31 @@ MatrixWorkspace::~MatrixWorkspace() {
   }
 }
 
+void MatrixWorkspace::setIndexTranslator(
```

Comment:
Yes, as you say, this is also a possible and good option for us. I think that forward declarations rather than includes are probably the better option when it comes to such a well used interfaces as `MatrixWorkspace`. I think we can live with having both signatures and maybe refactoring the one above out as part of the second step?